cloudformation/ecs-conex.template.js

```diff
@@ -58,6 +60,11 @@ var conex = {
       Description: '[secure] npm access token used to install private packages',
       Default: ''
     },
+    RepositoryPermissionPolicy: {
+      Type: 'String',
+      Description: 'An IAM policy granting access for other AWS accounts to access this repository',
```

Comment:
Really this policy could be granting _anything_ on the repositories that conex creates. Its just your particular use-case for cross-account access, right?